packages/block-library/src/site-title/edit.js

```diff
@@ -109,26 +113,52 @@ export default function SiteTitleEdit( {
 				/>
 			</BlockControls>
 			<InspectorControls>
-				<PanelBody title={ __( 'Settings' ) }>
-					<ToggleControl
-						__nextHasNoMarginBottom
+				<ToolsPanel
+					label={ __( 'Settings' ) }
+					resetAll={ () => {
+						setAttributes( {
+							isLink: false,
+							linkTarget: '_self',
+						} );
+					} }
+				>
+					<ToolsPanelItem
+						hasValue={ () => isLink !== false }
```

Comment:
The default value for `isLink` is `true`, so having a value means that the value is `false`.

Therefore, the boolean value here should be reversed, or this setting won't reset properly.

Reply:
Hi @t-hamano, Thanks for pointing out the issue. I've raised a pull request to address this. Please take a look at it when you have time.